src/cr/cube/dimension.py

```diff
@@ -882,11 +882,24 @@ def _iter_valid_subtotal_dicts(self):
                 continue
 
             # ---skip any malformed subtotal-dicts---
-            if not {"anchor", "args", "name"}.issubset(insertion_dict.keys()):
+            if not {"anchor", "name"}.issubset(insertion_dict.keys()):
+                continue
+
+            # ---use "new style" kwargs defining positive terms if available---
+            positive = insertion_dict.get("kwargs", {}).get("positive", [])
+
+            # ---but if missing, check for "old style" args defining positive terms---
+            if not positive:
+                positive = insertion_dict.get("args", [])
```

Comment:
```suggestion
            positive = insertion_dict.get("kwargs", {}).get(
                "positive"
            ) or insertion_dict.get("args", [])
```

Reply:
if you like this syntax you can replace the if and the positive assignment into 1 single line